src/test/java/com/wiley/autotest/framework/pages/TestElementPage.java

```diff
@@ -81,4 +79,32 @@ public TestElementPage checkNonExistingElementClickFails() {
         element(By.cssSelector("not_present_element"), new SearchStrategy(1)).click();
         return this;
     }
+
+    public TestElementPage checkSingleElementHasAnyText() {
+        element(By.id("exist")).should().haveText();
+        return this;
+    }
+
+    public TestElementPage checkFewElementsHasAnyText() {
+        elements(By.className("someClassForExistElements")).should().haveText();
+        return this;
+    }
+
+    public TestElementPage checkSingleElementHasNotAnyText() {
+        try {
+            domElement(By.id("notExist")).should().haveText();
+        } catch (AssertionError e) {
+            e.printStackTrace();
+        }
+        return this;
+    }
+
+    public TestElementPage checkNotAllElementsHasAnyText() {
+        try {
+            domElements(By.className("someClassForOneNotExistElements")).should().haveText();
+        } catch (AssertionError e) {
+            e.printStackTrace();
```

Comment:
what does this method check? I don't see any assertion in the catch. What if we get there?